Podfile

```diff
@@ -1,14 +1,14 @@
-platform :ios, '10.0'
+platform :ios, '11.0'
 use_frameworks!
 
 workspace 'AEPCampaignClassic'
 project 'AEPCampaignClassic.xcodeproj'
 
-pod 'SwiftLint', '0.44.0'
+pod 'SwiftLint', '0.52.0'
 
 def campaignclassic_dependencies
-   pod 'AEPCore'
-   pod 'AEPServices'
+   pod 'AEPCore', '~> 4.0.0'
+   pod 'AEPServices', '~> 4.0.0'
```

Comment:
This will only get you up to 4.1 (i.e. version < 4.1). Did you intend to use `~> 4.0` which will allow v < 5.0.

Reply:
updated to use `~> 4.0`